### PR TITLE
Add support for compilation with quicklz

### PR DESCRIPTION
**Pull request by asubramanya**

### CODE REVIEW

File: concourse/scripts/compile_gpdb.bash [hunk 1]
```diff
@@ -138,6 +138,12 @@ function _main() {
   else
     BLD_TARGET_OPTION=("")
   fi
+
+  # Copy gpaddon_src into gpAux/addon directory and set the ADDON_DIR
+  # environment variable, so that quicklz support is available in enterprise
+  # builds.
+  export ADDON_DIR=addon
+  cp -R gpaddon_src gpdb_src/gpAux/$ADDON_DIR
```
Comment (ashwinstar) [inline concourse/scripts/compile_gpdb.bash]: Does symbolic link not work ?

Comment (asubramanya) [inline concourse/scripts/compile_gpdb.bash]: Symlinking doesn't seem to work. I have added a comment in the bash script.

**Comment by asimrp** [comment]
@asubramanya anything needs to be done here?

**Comment by asubramanya** [comment]
Thanks for the review. Pushed: ba1d4c0a9e1206a2943a0431266d45c20c10beb3

**Pull request closed without merging**